include/sound/sof/dai.h

```diff
@@ -69,7 +69,8 @@ struct sof_ipc_dai_config {
 
```

Comment:
@plbossart I object to the "somehow" wording in commit message. We explicitly agreed in the ABI process that it's not mandatory to update the kernel-side fields until the feature is taken to use on kernel side:
https://thesofproject.github.io/latest/contribute/process/abiprocess.html

Reply:
@kv2019i my point is that this group_id feature is not really an ABI change if we add fields that the kernel will never use...

Reply:
@plbossart I'm completely ok with the change. Just commit on the "somehow" part. It's no perfectly ok (process wise) to modify the FW side ABI headers and not send a matching kernel patch (as long as the change is backwards compatible). So in this sense it's normal that some structs are not aligned (kernel is stuck to an older ABI).